src/vs/workbench/contrib/terminal/browser/remoteTerminalService.ts

```diff
@@ -190,7 +190,8 @@ export class RemoteTerminalService extends Disposable implements IRemoteTerminal
 		return this._remoteTerminalChannel.setTerminalLayoutInfo(layout);
 	}
 
-	public getTerminalLayoutInfo(): Promise<ITerminalsLayoutInfo | undefined> {
+	public async getTerminalLayoutInfo(): Promise<ITerminalsLayoutInfo | undefined> {
+		await this._remoteTerminalChannel?.listProcesses(true);
```

Comment:
:+1: we no longer use `listProcesses` for this so it no longer needs it. We could also just add a `reduceConnectionGraceTime` method and not await it, instead of integrating it into `getTerminalLayoutInfo` and maybe changing it again later.

Reply:
https://github.com/microsoft/vscode/issues/118724